components/navigation/api/src/main/resources/ApplicationResources.properties

```diff
@@ -69,6 +69,8 @@ phenotips.allData.menu.export.excel=Export as Excel file
 phenotips.allData.menu.export.excel.tooltip=Export selected patients in MS Excel format
 phenotips.allData.menu.export.json=Export as JSON
 phenotips.allData.menu.export.json.tooltip=Export selected patients in JSON format
+phenotips.allData.menu.import.json=Import patients from JSON
```

Comment:
The layout of the dialog looks confusing. Please adjust the styling to obtain the look in this screenshot:

![9d598a3a-a0fc-11e6-953e-fb80af6ba91b](https://cloud.githubusercontent.com/assets/88663/19941816/f2e069c8-a107-11e6-90ce-53aabcff6394.png)